test/documentation/api-stability.js

```diff
@@ -13,12 +13,21 @@ $(document).ready(function()
         $(this).find('div.memproto').html(new_html);
     }
 
-    var add_labels = function(api_type, entry_type)
+    var add_labels = function(api_type, entry_obj, entry_type)
```

Comment:
@eseiler Last question, can we get rid of `entry_obj` and use `this`?

Reply:
I'll check